mern-ecommerce/controllers/auth.js

```diff
@@ -62,20 +62,20 @@ exports.requireSignin = expressJwt({
   userProperty: 'auth',
 });
 
-exports.isAuth = (req, res, next) => {
-  let user = req.profile && req.auth && req.profile._id == req.auth._id;
-  if (!user) {
+exports.isAdmin = (req, res, next) => {
+  if (req.profile.role === 0) {
     return res.status(403).json({
-      error: 'Access denied',
+      error: 'Admin resource! Access denied',
     });
   }
   next();
 };
 
-exports.isAdmin = (req, res, next) => {
-  if (req.profile.role === 0) {
+exports.isAuth = (req, res, next) => {
+  let user = req.profile && req.auth && req.profile._id == req.auth._id;
+  if (!user) {
     return res.status(403).json({
-      error: 'Admin resource! Access denied',
+      error: 'Access denied',
```

Comment:
The error message 'Access denied' is not internationalized. It would be better to use a translation function or a separate file for error messages to make the application more scalable and maintainable.